test/screen/RequestScreen.js

```diff
@@ -170,63 +189,82 @@ describe('RequestScreen', function() {
 				clearTimeout(id);
 				done();
 			});
-		var id = setTimeout(() => this.requests[0].respond(200), 100);
 	});
 
 	it('should fail for invalid status code response', (done) => {
+		fetchStub.returns(Promise.resolve(
+			new Response('', {status: 404})
+		));
+
 		new RequestScreen()
 			.load('/url')
 			.catch((error) => {
 				assert.ok(error.invalidStatus);
 				done();
 			});
-		this.requests[0].respond(404);
 	});
 
 	it('should return the correct http status code for "page not found"', (done) => {
+		fetchStub.returns(Promise.resolve(
+			new Response('', {status: 404})
+		));
+
 		new RequestScreen()
 			.load('/url')
 			.catch((error) => {
 				assert.strictEqual(error.statusCode, 404);
 				done();
 			});
-		this.requests[0].respond(404);
 	});
 
 	it('should return the correct http status code for "unauthorised"', (done) => {
+		fetchStub.returns(Promise.resolve(
+			new Response('', {status: 401})
+		));
+
 		new RequestScreen()
 			.load('/url')
 			.catch((error) => {
 				assert.strictEqual(error.statusCode, 401);
 				done();
 			});
-		this.requests[0].respond(401);
 	});
 
-
 	it('should fail for request errors response', (done) => {
+		fetchStub.returns(Promise.reject(
+			new Error(errors.REQUEST_ERROR)));
+
 		new RequestScreen()
 			.load('/url')
 			.catch((error) => {
 				assert.ok(error.requestError);
 				done();
 			});
-		this.requests[0].error();
 	});
 
 	it('should form navigate force post method and request body wrapped in FormData', (done) => {
 		globals.capturedFormElement = globals.document.createElement('form');
+
+		let formData = new FormData(globals.capturedFormElement);
```

Comment:
This seems to be unused? Or maybe the GitHub UI is blinding me...

Reply:
It was unused yes.